### ISSUE TITLE
I can't follow the tutorial to enable http3

### ISSUE THREAD

**Issue by expoli** [traefik/traefik#8095]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

On this page, I see that traefik now supports HTTP3, the URL is https://doc.traefik.io/traefik/master/routing/entrypoints/#enablehttp3
But when I want to enable this feature, traefik said 
`command traefik error: failed to decode configuration from flags: field not found, node: http3`
I checked the commit and find that http3 support has been merged to the master branch, you can see that at this page: https://github.com/traefik/traefik/pull/7724 . 

I use docker to run traefik and the image tag is latest, you can find it at this page https://hub.docker.com/layers/traefik/library/traefik/latest/images/sha256-3ac00a4751157d604689d4dd0e880cbdba19eaeee7105e1a811c735a0d152465?context=explore .

If I want to enable HTTP3 in tarefik then what should I do ?

### What did you expect to see?

Enable HTTP3 in tarefik 

### What did you see instead?

`traefik    | 2021/04/27 05:21:14 command traefik error: failed to decode configuration from flags: field not found, node: http3`

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.4.8
Codename:     livarot
Go version:   go1.16.2
Built:        2021-03-23T15:48:39Z
OS/Arch:      linux/arm64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```yaml
version: "3.3"

services:

  traefik:
    image: "traefik"
    container_name: "traefik"
    restart: always
    command:
      - "--log.level=DEBUG"
      - "--api=true"
      - "--providers.docker=true"
      - "--providers.docker.exposedbydefault=false"
      - "--entrypoints.web.address=:80"
      - "--entrypoints.websecure.address=:443"
      - "--certificatesresolvers.myresolver.acme.httpchallenge=true"
      - "--certificatesresolvers.myresolver.acme.httpchallenge.entrypoint=web"
      # - "--certificatesresolvers.myresolver.acme.caserver=https://acme-staging-v02.api.letsencrypt.org/directory"
      - "--certificatesresolvers.myresolver.acme.email="
      - "--certificatesresolvers.myresolver.acme.storage=/letsencrypt/acme.json"
      - "--metrics.prometheus=true"
      - "--entryPoints.metrics.address=:8082"
      - "--metrics.prometheus.entryPoint=metrics"
      - "--entrypoints.web.http.redirections.entryPoint.to=websecure"
      - "--entrypoints.web.http.redirections.entryPoint.scheme=https"
      - "--experimental.http3=true"
      - "--entrypoints.websecure.enablehttp3=true"
    # ports:
    #   - "80:80"
    #   - "443:443"
    #   - "8080:8080"
    network_mode: host
    volumes:
      - "./letsencrypt:/letsencrypt"
      - "/var/run/docker.sock:/var/run/docker.sock:ro"

  whoami:
    image: "traefik/whoami"
    container_name: "simple-service"
    restart: always
    labels:
      - "traefik.enable=true"
      - "traefik.http.routers.whoami.rule=Host(``)"
      - "traefik.http.routers.whoami.entrypoints=websecure"
      - "traefik.http.routers.whoami.tls.certresolver=myresolver"

    networks:
      - proxy-tier

networks:
  proxy-tier:
    external:
      name: traefik
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
traefik    | 2021/04/27 05:21:14 command traefik error: failed to decode configuration from flags: field not found, node: http3
```

**Comment by expoli** [comment]
https://github.com/traefik/traefik/pull/7724#issuecomment-763425614

this commit said http3 feature will released in the next version of Traefik (v2.5), if I want to use this feature Is it possible to compile the master branch source code?

**Comment by jbdoumenjou** [comment]
Hello @expoli,

Indeed, to use this feature, you have to [compile](https://doc.traefik.io/traefik/master/contributing/building-testing/) from the master branch.

For help on your configuration, please join our [Community Forum](https://community.traefik.io/) and reach out to us on the [Traefik section](https://community.traefik.io/c/traefik). :smiley: 

So, I close this issue as a question.
